server/constants/constants.go

```diff
@@ -92,16 +92,17 @@ const (
 	QueryParamPerPage     = "per_page"
 
 	// Filters
-	FilterCreatedByMe            = "me"
-	FilterCreatedByAnyone        = "anyone"
-	FilterAllChannels            = "all_channels"
-	FilterAll                    = "all"
-	FilterBoards                 = "boards"
-	FilterRepos                  = "repos"
-	FilterPipelines              = "pipelines"
-	EventTypeRelease             = "release"
-	FilterIDReleaseDefinitionID  = "releaseDefinitionId"
-	FilterIDReleaseEnvironmentID = "releaseEnvironmentId"
+	FilterCreatedByMe          = "me"
+	FilterCreatedByAnyone      = "anyone"
+	FilterAllChannels          = "all_channels"
+	FilterAll                  = "all"
+	FilterBoards               = "boards"
+	FilterRepos                = "repos"
+	FilterPipelines            = "pipelines"
+	FilterReleaseDefinitionID  = "releaseDefinitionId"
+	FilterReleaseEnvironmentID = "releaseEnvironmentId"
+
+	EventOfTypeRelease = "release"
```

Comment:
i think the previous name was better

Reply:
I changed this name because there is no eventType `release` but there are events which contain `release` in their name, that's what we are checking with this constant